README.md

```diff
@@ -1,7 +1,7 @@
 # protocol-substrate
 ### Build
 
-The `cargo run` command will perform an initial build. Use the following command to build the node
+Clone the submodules using `git submodule update --init`. The `cargo run` command will perform an initial build. Use the following command to build the node
```

Comment:
maybe worth mentioning that if you are first-time cloning you can do it in just one step or one call like so:

```
git clone --recurse-submodules -j8 https://github.com/webb-tools/protocol-substrate.git
```